local-test.js

```diff
@@ -0,0 +1,43 @@
+const fs = require('fs')
+const Firestore = require('./utils/firestore.js')
+const config = require('config')
+
+// check whether github credentials are not falsy
+if (
+  !config.githubOauth ||
+  !config.githubOauth.clientId ||
+  !config.githubOauth.clientSecret
+) {
+  throw new Error('Github credentials are not properly set')
+}
+
+// throw an error if unable to read file
+try {
+  var firestoreData = fs.readFileSync('./firestore-private-key.json', 'utf8')
```

Comment:
Can we use const over here, since the variable is not changing?

Reply:
var can't be access outside  of block scope.
If i will use const then variable will not be accessed for if condition on line 22.

If i declare a const variable but doesn't initialize then i will not be able to assign other value to the variable.
So, var is instead of const.